workers/data_refinery_workers/downloaders/utils.py

```diff
@@ -78,15 +79,14 @@ def queue_task(processor_job):
 
     if success:
         for batch in batches:
-            with transaction.atomic():
-                processor_job = save_batch_create_job(batch)
-                if batch.pipeline_required != ProcessorPipeline.NONE.value:
-                    success = queue_task(processor_job)
-                    if success:
-                        logger.info("Downloader Job %d completed successfully.", job.id)
-                    else:
-                        failure_template = "Could not find Processor Pipeline {} in the lookup."
-                        job.failure_reason = failure_template.format(batch.pipeline_required)
+            processor_job = save_batch_create_job(batch)
```

Comment:
I moved the transaction to only be around the job-creation call because including the message in the transaction can cause some weird behavior. I've created an issue to just do this within that function instead of wrapping every invocation of it: https://github.com/data-refinery/data_refinery/issues/45